### DISCUSSION TITLE
There was an error loading your app

### DISCUSSION THREAD

**Discussion by syeillah** [Budibase/budibase#5578] [Help]
Hi, may i know the root cause for this case?
I try to open/ preview my app then this happen, also the attachment section on internal db budibase are crack. 

<img width="1435" alt="Screen Shot 2022-04-26 at 13 41 41" src="https://user-images.githubusercontent.com/99949174/165238178-88735d46-82e6-4acd-96ee-df4a7a7ae5f1.png">
<img width="623" alt="Screen Shot 2022-04-26 at 13 43 19" src="https://user-images.githubusercontent.com/99949174/165238191-3e46e123-718b-4c2a-b4f2-55f61a73c276.png">

**Comment by melohagan** [comment]
HI @syeillah 

If you login to your minio instance, can you see if the attachments are still there for your app? 

Are all of the services running? Have you tried restarting them?

As you are self hosting, maybe you could try a re-install.
